lib/src/response/responder.rs

```diff
@@ -201,6 +201,16 @@ impl Responder<'static> for String {
     }
 }
 
+/// Returns a response with a fixed-size body
+/// containing the byte vec `self`. Always returns `Ok`.
+impl Responder<'static> for Vec<u8> {
```

Comment:
Why do we need this? Why can't we just use `Cursor::new(vec)` in the `MsgPack` `Responder`?